### ISSUE TITLE
be more specific about the underlying IEEE-754 operation in numeric operation algorithm steps

### ISSUE THREAD

**Issue by michaelficarra** [tc39/ecma262#1988]
For context, see https://github.com/tc39/ecma262/pull/1974#discussion_r416988843.

**Comment by bakkot** [comment]
@michaelficarra https://github.com/tc39/ecma262/pull/2189 would resolve this, yes?

**Comment by michaelficarra** [comment]
@bakkot I don't think it changes the situation. I think we still need to do an audit to see where we do and don't think it's appropriate to mention 754.
